ada/language/ast.py

```diff
@@ -6379,6 +6379,11 @@ def xref_equation():
             # entry.
             Entity.id.name_is('Global'), Entity.expr.sub_equation,
 
+            # Do not resolve anything inside those aspect, as identifiers act
```

Comment:
This won't work for `pragma Convention` right ?

Reply:
`pragma Convention` was already handled (see `Pragma.xref_equation`)